pom.xml

```diff
@@ -45,19 +45,26 @@
   <properties>
     <!--Project properties-->
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
     <compiler.fork>false</compiler.fork>
-    <maven.deploy.plugin.version>2.8.2</maven.deploy.plugin.version>
+    <maven.bundle.plugin.version>4.2.1</maven.bundle.plugin.version>
+    <maven.compiler.plugin.version>3.8.1</maven.compiler.plugin.version>
+    <maven.deploy.plugin.version>3.0.0-M1</maven.deploy.plugin.version>
+    <maven.enforcer.plugin.version>3.0.0-M3</maven.enforcer.plugin.version>
     <maven.release.plugin.version>3.0.0-M1</maven.release.plugin.version>
+    <maven.resources.plugin.version>3.2.0</maven.resources.plugin.version>
+    <maven.surefire.plugin.version>3.0.0-M5</maven.surefire.plugin.version>
     <javalin.thirdparty.bundle.version>2.5.0_1</javalin.thirdparty.bundle.version>
     <kotlin-osgi.version>1.2.61</kotlin-osgi.version>
-    <maven-jar-plugin.version>3.0.2</maven-jar-plugin.version>
-    <errorprone.version>2.2.0</errorprone.version>
+    <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
+    <errorprone.version>2.4.0</errorprone.version>
+    <jacoco.maven.plugin.version>0.8.6</jacoco.maven.plugin.version>
+    <fmt.maven.plugin.version>2.3.0</fmt.maven.plugin.version>
 
     <!--Backend properties-->
-    <ddf.version>2.26.5</ddf.version>
-    <ddf-jsonrpc.version>0.6</ddf-jsonrpc.version>
+    <ddf.version>2.27.0-SNAPSHOT</ddf.version>
```

Comment:
java 11 ddf hasn't been released yet

Reply:
That's a good point, we probably should plan on a DDF release sometime soon. Maybe after a few more upgrades so we can release the whole DDF stack